src/addons/addons_manager.cpp

```diff
@@ -583,13 +583,19 @@ bool AddonsManager::uninstall(const Addon &addon)
  */
 void AddonsManager::saveInstalled()
 {
-    //Put the addons in the xml file
-    //Manually because the irrlicht xml writer doesn't seem finished, FIXME ?
+    // Put the addons in the xml file
+    // Manually because the irrlicht xml writer doesn't seem finished, FIXME ?
     std::ofstream xml_installed(m_file_installed.c_str());
 
-    //write the header of the xml file
+    // Write the header of the xml file
     xml_installed << "<?xml version=\"1.0\"?>" << std::endl;
-    xml_installed << "<addons  xmlns='http://stkaddons.net/'>"
+
+    // Get server address from config
+    std::string server = UserConfigParams::m_server_addons;
+    // Find the third slash (end of the domain)
+    std::string::size_type index = server.find('/');
+    index = server.find('/', index + 2) + 1; // Omit one slash
```

Comment:
This is not that portable. Optimally you should find the last slash. If the domain is: http://example.com/somefolder/.